lib/livebook_web.ex

```diff
@@ -79,7 +79,8 @@ defmodule LivebookWeb do
 
       # We don't know the hostname Livebook runs on, so we don't use
       # absolute URL helpers
```

Comment:
```suggestion
      # We don't know the hostname Livebook runs on, so we don't use
      # absolute URL helpers. We don't import sigil_p either, because
      # we override it.
```